packages/contracts/src/networks.ts

```diff
@@ -1,104 +1,27 @@
-import {
-  AccountAccess,
-  AccountAccessKind,
-  DeployedContractSize,
-  ParsedAccountAccess,
-} from './types'
+import { DeployedContractSize, ParsedAccountAccess } from './types'
 
 /**
- * Calculates the storage used by a single AccountAccess. This function is intentionally very simple.
- * We naively assume that every write takes up a full 32 bytes of storage despite the fact that there
- * are many cases where the storage usage is less or even negative. We do this because we prefer to
- * always overestimate the cost by a reasonable amount. Since the block gas limit on Moonbeam and
- * related networks is comfortably high (15 million), this does not impede the users ability to deploy
- * large contracts.
- */
-const calculateStorageSizeForAccountAccess = (
-  access: AccountAccess,
-  deployedContractSizes: DeployedContractSize[]
-): { contractStorageSize: number; writeStorageSize: number } => {
-  const storageWriteSize =
-    access.storageAccesses.filter((storageAccess) => storageAccess.isWrite)
-      .length * 32
-
-  if (access.kind === AccountAccessKind.Create) {
-    const deployedContractSize = deployedContractSizes.find(
-      (deployedContact) => deployedContact.account === access.account
-    )
-
-    if (!deployedContractSize) {
-      throw new Error(
-        'Failed to find deployed contract size. This is a bug, please report it to the developers.'
-      )
-    }
-
-    return {
-      writeStorageSize: storageWriteSize,
-      contractStorageSize: Number(deployedContractSize.size),
-    }
-  } else {
-    return {
-      writeStorageSize: storageWriteSize,
-      contractStorageSize: 0,
-    }
-  }
-}
-
-/**
- * Calculates the cost of a transaction on Moonbeam using their higher gas cost per byte.
+ * Return a hard-coded gas value for Moonbeam, Moonbase Alpha, and Moonriver. We hard-code this
+ * value as a temporary solution because these networks have a non-standard gas calculation
+ * mechanism.
  *
  * @param baseGas The estimated gas cost according to Foundry.
  * @param deployedContractSizes The sizes of any contracts deployed during this transaction.
  * @param access The ParsedAccountAccess for the transaction.
  * @returns
  */
 export const calculateActionLeafGasForMoonbeam = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   foundryGas: string,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   deployedContractSizes: DeployedContractSize[],
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   access: ParsedAccountAccess
 ): string => {
```

Comment:
In the interest of time, this is fine, but we should clean these up if they don't get used in the longer term fix for this.